### DISCUSSION TITLE
$auth.loggedIn and user always return false

### DISCUSSION THREAD

**Discussion by Owura-Kwaku** [nuxt-community/auth-module#1554] [Support Requests]
### Version
module: @nuxtjs/auth-next 2.14.05.0.0-1624817847.21691f1
nuxt: 2.14.0

#### [mode](https://nuxtjs.org/api/configuration-mode): 
 - [x] universal
 - [ ] spa

### Nuxt configuration
here is my nuxt.config.js

```
  auth: {
    strategies: {
      google: {
        clientId:
          ' 941383346878-q2j64498ic9hubn079hb2j5or49cjvg3.apps.googleusercontent.com'
      },
      facebook: {
        endpoints: {
          userInfo:
            'https://graph.facebook.com/v6.0/me?fields=id,name,picture{url}'
        },
        clientId: '574492650390644',
        scope: ['public_profile', 'email']
      },
      cookie: {
        scheme: 'cookie',
        cookie: {
          name: 'api-eskool'
        },
        user: {
          property: false,
          autoFetch: true
        },
        endpoints: {
          login: {
            url: '/login',
            method: 'post'
          },
          logout: {
            url: '/login',
            method: 'delete'
          },
          user: {
            url: '/profile',
            method: 'get'
          }
        }
      }
    }
  },

  axios: {
    baseURL: 'https://api.myeskool.com/users'
  }
```

#### What is expected?
When the user logs in `$auth.loggedIn` must return a true value and `user` must return an object

#### What is actually happening?
After user logs in, `$auth.loggedIn` and `user` are always false

## Checklist
* [ ]  I have tested with the latest Nuxt version and the issue still occurs
* [ ]  I have tested with the latest module version and the issue still occurs
* [x]  I have searched the issue tracker and this issue hasn't been reported yet

**Comment by rolexta** [comment]
same issue with "@nuxtjs/auth-next": "5.0.0-1624817847.21691f1"
